src/test/java/com/google/api/codegen/testdata/python_types_no_path_templates.baseline

```diff
@@ -0,0 +1,39 @@
+============== file: example/types.py ==============
```

Comment:
Explained offline so I'll be brief here. no_path_templates tests unversioned APIs like longrunning. This is why the types are generated to this namespace.